modules/experimental-layers/src/gpu-grid-layer/gpu-grid-cell-layer.js

```diff
@@ -31,7 +31,7 @@ const DEFAULT_MAXCOLOR = [0, 255, 0, 255];
 const AGGREGATION_DATA_UBO_INDEX = 0;
 
 const defaultProps = {
-  cellSize: {type: 'number', min: 0, max: 20, value: 20},
+  cellSize: {type: 'number', min: 0, max: 1000, value: 1000},
```

Comment:
@1chandu is there a reason to limit `cellSize` to 20?

Reply:
Nope, there shouldn't be any, this change is fine.